src/components/AppHeader/AppHeader.js

```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import './AppHeader.scss';
+import Logo from '../Logo/Logo';
```

Comment:
you could use the index.js strategy instead

Reply:
won't change all right now but indexes added